go.mod

```diff
@@ -5,25 +5,25 @@ go 1.20
 require (
 	github.com/getsentry/sentry-go v0.25.0
 	github.com/rs/zerolog v1.29.1
-	k8s.io/api v0.25.12
-	k8s.io/apimachinery v0.25.12
-	k8s.io/client-go v0.25.12
+	k8s.io/api v0.28.4
+	k8s.io/apimachinery v0.28.4
+	k8s.io/client-go v0.28.4
```

Comment:
If it can be avoided -- let's not bump `client-go`, newer versions and features/APIs might not be fully compatible with older k8s clusters.